integration-tests/parse_files.sh

```diff
@@ -109,8 +109,11 @@ function parse_dockerfiles() {
 	if [[ "$BLACKLIST" == *$dockerfile* ]]; then
 		continue;
 	fi
-        if ./parseFile "$dockerfile" | grep -a1 unexpected; then
-            result="false"
+
+        if ./parseFile "$dockerfile" > /dev/null; then
+	    echo "$dockerfile"
+        else
+	    result="false"
```

Comment:
![image](https://user-images.githubusercontent.com/18702153/41064144-9075c7ac-69d2-11e8-9cd9-5e4e887db3f5.png)
Can you please fix indentation? Some of them are tabs and some spaces.

Reply:
I would prefer spaces if you don't mind.